src/main/java/io/jenkins/plugins/view/calendar/service/CronJobService.java

```diff
@@ -103,10 +113,8 @@ public List<CronTab> getCronTabs(final Trigger trigger, final Hash hash) {
     @SuppressWarnings("PMD.CyclomaticComplexity")
     public List<Trigger> getCronTriggers(final Job job, final CalendarViewEventsType eventsType) {
         final Collection<Trigger<?>> jobTriggers;
-        if (job instanceof AbstractProject) {
```

Comment:
The distinction between `AbstractProject` and `WorkflowJob` is unnecessary as both implement `TriggeredItem`. That would also ensure that any new job type that inherits directly from `Job` and is a `TriggeredItem` would be detected.

Reply:
You don't have to make a separate PR out of it, I'm fine with it being part of this one.